test/client/bst-config-test.ts

```diff
@@ -66,12 +66,19 @@ describe("BSTConfig", function() {
 describe("BSTProcess", function() {
     describe("#run()", function() {
         let sandbox: SinonSandbox = null;
+
+        before(function () {
+            (<any> BSTProcess).processPath = function () {
```

Comment:
Perhaps `BSTProcess` should type asserted to [`class BSTProcess`](https://github.com/bespoken/bst/blob/67cee7462258807a8420e1d967912c479eca5ec7/lib/client/bst-config.ts#L102) ? — same in line [154](https://github.com/bespoken/bst/pull/235/commits/457a8727a5957845512c498848850a080b382529#diff-2ae95d89409189e7a5d7f6c3307fc8f4R154)